script/base.ps1

```diff
@@ -0,0 +1,66 @@
+# Install Chocolatey if not already installed
+if (-not (Get-Command choco -ErrorAction SilentlyContinue)) {
+    Set-ExecutionPolicy Bypass -Scope Process -Force
+    $installDir = Join-Path $env:USERPROFILE "choco"
+    Invoke-Expression ((New-Object System.Net.WebClient).DownloadString("https://chocolatey.org/install.ps1")) -installDirectory $installDir
+}
+
+# Install Git if not already installed
+if (-not (Get-Command git -ErrorAction SilentlyContinue)) {
+    choco install git -y
+}
+
+# Update and upgrade system
+choco upgrade all -y
+
+# Install docker for containers and microservices
+# https://docs.docker.com/engine/install/ubuntu/
+choco install -y docker-desktop
```

Comment:
better to check for the existence of docker-desktop before installing the same. Lines 19 to 42 need to be wrapped into that checking block.

Reply:
Line-19 throws an error if docker-desktop is already installed.